custodia/httpd/server.py

```diff
@@ -21,6 +21,14 @@
     from socketserver import ForkingTCPServer
     from urllib.parse import urlparse, parse_qs, unquote
 
+try:
+    # pylint: disable=import-error
+    from systemd.daemon import notify as sd_notify
+    from systemd.daemon import booted as systemd_booted
```

Comment:
can we use a consistent naming here ?
either sd_notify, sd_booted or systemd_notify, systemd_booted work for me.